pkg/plugins/manager/signature/manifestverifier/verifier.go

```diff
@@ -30,18 +37,82 @@ type ManifestVerifier struct {
 	cfg  *config.Cfg
 	mlog log.Logger
 
-	lock       sync.Mutex
-	cli        http.Client
-	publicKeys map[string]ManifestKeys
+	lock sync.Mutex
+	cli  http.Client
+	kv   plugins.KeyStore
+}
+
+func New(cfg *config.Cfg, mlog log.Logger, kv plugins.KeyStore) *ManifestVerifier {
+	pmv := &ManifestVerifier{
+		cfg:  cfg,
+		mlog: mlog,
+		cli:  makeHttpClient(),
+		kv:   kv,
+	}
+	return pmv
 }
 
-func New(cfg *config.Cfg, mlog log.Logger) *ManifestVerifier {
-	return &ManifestVerifier{
-		cfg:        cfg,
-		publicKeys: map[string]ManifestKeys{},
-		mlog:       mlog,
-		cli:        makeHttpClient(),
+func (pmv *ManifestVerifier) Run(ctx context.Context) error {
+	if !pmv.featureEnabled() {
```

Comment:
We should probably implement https://github.com/grafana/grafana/blob/main/contribute/backend/services.md#disabled-services. But since the registered background service is not this service, rather the `signature.Signature` maybe we should implement `service.IsDisabled ` for `signature.Signature` and remove this feature toggle check?